### PR TITLE
SZ:

### PR DESCRIPTION
**Pull request by numismatika**
- re-enable ruleset

### CODE REVIEW

File: src/chrome/content/rules/Suddeutsche_Zeitung.xml [hunk 1]
```diff
@@ -72,7 +66,6 @@ Fetch error: http://www.sueddeutsche.de/favicon.ico => https://www.sueddeutsche.
 		- maerkte
 		- motormarkt
 		- service
-		- stellenmarkt
```
Comment (J0WI) [inline src/chrome/content/rules/Suddeutsche_Zeitung.xml]: Please move it to the `Nonfunctional domains` section.

File: src/chrome/content/rules/Suddeutsche_Zeitung.xml [hunk 2]
```diff
@@ -153,7 +147,6 @@ Fetch error: http://www.sueddeutsche.de/favicon.ico => https://www.sueddeutsche.
 	<test url="http://geoservice.sueddeutsche.de/" />
 	<test url="http://global.sueddeutsche.de/assets/img/header/sprite_header.png" />
 	<test url="http://id.sueddeutsche.de/" />
-	<test url="http://jetzt.sueddeutsche.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Suddeutsche_Zeitung.xml]: Please move it to the `Nonfunctional` or `Problematic` domains section.

Comment (numismatika) [reply src/chrome/content/rules/Suddeutsche_Zeitung.xml]: it is gone. The youth magazine lives on www.jetzt.de

Comment (J0WI) [reply src/chrome/content/rules/Suddeutsche_Zeitung.xml]: The old host is still alive from here, but times out on https.

**Pull request merged**